tutorials/applications/Bayesian Optimisation for antiferromagnetic state preparation.ipynb

```diff
@@ -22,7 +22,7 @@
    "cell_type": "markdown",
```

Comment:
<p>Can get rid of "an an interpolation function PchipInterpolator since it's in Pulser now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>